app/src/tests/mocks/mockResponses.ts

```diff
@@ -0,0 +1,30 @@
+export const processFilesMockResponseNonConflict = {
```

Comment:
Hmm, I would have expected a single type to be defined for the response.  Then uploadFiles() could return Observable\<returnType\> rather than Observable\<any\>.

Reply:
Rafi and I were thinking of having different types of responses depending on whether or not there are merge conflicts that the user must resolve. However, this may not the best approach.